src/pages/index.tsx

```diff
@@ -1,5 +1,38 @@
 import CampaignHero from "../components/hero/campaignHero";
+import {
+  SkipNavigation,
+  Template,
+  TemplateAboveHeader,
+  TemplateBreakout,
+  TemplateContent,
+  TemplateFooter,
+  TemplateHeader,
+} from "@nypl/design-system-react-components";
 
 export default function Home() {
-  return <CampaignHero />;
+  return (
+    <>
+      <SkipNavigation />
```

Comment:
Thanks for adding this, I forgot about this. It also needs a bit more clarity from Clare or Lana about where the "Skip to Main Content" link should go to. The link goes to `#mainContent` but there's no element on the page with this id. Can you add it to the h1 in the Hero (currently has `campaign-hero`) for now?

We can bring this up once the swim lanes are complete -- should we point to the hero heading or the first swim lane?